### PR TITLE
TO-TDAP-426-Update-the-TestOps-readme-page

### PR DESCRIPTION
**Pull request by haolekatalon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,43 @@
-# Katalon TestOps
+<h1 align="center">
+<a href="https://www.katalon.com/testops/" target="blank_">
+    <img width=70% src="https://github.com/katalon-studio/docs-images/raw/master/katalon-analytics/product-logo/Core_TestOps.png" alt="Katalon TestOps Logo" />
+</a>
+</h1>
 
-[Katalon TestOps](https://analytics.katalon.com) is a web-based application that provides dynamic perspectives and an insightful look at your automation testing data. You can leverage your automation testing data by transforming and visualizing your data; analyzing test results; seamlessly integrating with such tools as Katalon Studio and Jira; maximizing the testing capacity with remote execution.
+**Katalon TestOps** is a web-based application that provides dynamic perspectives and an insightful look at your automation testing data.
```
Comment (AntoineTouch) [inline README.md:7]: TestOps is not just a report generation tool, it is also an orchestration and remote execution tool, so this has to be in the blurb.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,43 @@
...
+For instructions on using Katalon TestOps, visit our documentation website: [Full documentation](https://docs.katalon.com/katalon-analytics/docs/overview.html).
+
+## Community and support
 
 * Read our [documentation](https://docs.katalon.com/katalon-analytics/docs/overview.html).
-* Ask a question on [Forum](https://forum.katalon.com/categories/katalon-analytics).
-* Request a new feature on [GitHub](CONTRIBUTING.md).
+* Learn more about orchestration of test automation and DevOps by enrolling in [Katalon Academy](https://academy.katalon.com/#katalon-testops) courses.
+* Ask questions on the [Katalon Community](https://forum.katalon.com/c/katalon-analytics) forum.
+* For community contribution, refer to our [contribution guidelines](CONTRIBUTING.md).
```
Comment (AntoineTouch) [inline README.md:28]: contributions

File: README.md [hunk 3]
```diff
@@ -1,9 +1,43 @@
...
+## Documentation
+
+For instructions on using Katalon TestOps, visit our documentation website: [Full documentation](https://docs.katalon.com/katalon-analytics/docs/overview.html).
+
+## Community and support
 
 * Read our [documentation](https://docs.katalon.com/katalon-analytics/docs/overview.html).
-* Ask a question on [Forum](https://forum.katalon.com/categories/katalon-analytics).
-* Request a new feature on [GitHub](CONTRIBUTING.md).
+* Learn more about orchestration of test automation and DevOps by enrolling in [Katalon Academy](https://academy.katalon.com/#katalon-testops) courses.
```
Comment (AntoineTouch) [inline README.md:26]: about test orchestration and DevOps

File: README.md [hunk 4]
```diff
@@ -1,9 +1,43 @@
...
+
+### Katalon TestCloud
+
+[Katalon TestCloud](https://www.katalon.com/testcloud/) is a cloud-based test execution environment where you can automate test scripts across the most updated browsers and operating systems. TestCloud saves time and reduces the cost of creating physical infrastructure to simulate the same platforms for test automation.
+
+### Katalon Recorder
+
+[Katalon Recorder](https://www.katalon.com/katalon-recorder-ide/) is a free and lightweight tool to kickstart test automation from zero without sacrificing the future. It helps you gain confidence in your release right away by allowing you to create and execute many tests quickly, efficiently and robustly. Later on, it helps you optimize testing activities with reusable test artifacts and the ability to categorize and execute tests flexibly. Down the road, it helps you scale without losing investment by providing easy migration paths to Katalon Studio and popular Selenium WebDriver frameworks + languages.
```
Comment (AntoineTouch) [inline README.md:43]: This can be shortened

**Review by AntoineTouch — CHANGES_REQUESTED** [review]

**Review by AntoineTouch — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -1,9 +1,42 @@
...
 
-[Katalon TestOps](https://analytics.katalon.com) is a web-based application that provides dynamic perspectives and an insightful look at your automation testing data. You can leverage your automation testing data by transforming and visualizing your data; analyzing test results; seamlessly integrating with such tools as Katalon Studio and Jira; maximizing the testing capacity with remote execution.
+**Katalon TestOps** is an enterprise-class platform for QA orchestration, test analytics, and advanced reports. Katalon TestOps provides dashboards and reports on the status of product quality.
+## Features
+
+With Katalon TestOps, you can:
+
+* Leverage automation testing by transforming and visualizing test data.
+* Analyze test results.
+* Maximize testing capacity with remote execution.
```
Comment (maile-katalon) [inline README.md:14]: With Katalon TestOps, you can:

* Maximize testing capacity with remote execution.
* Analyze and visualize test data in advanced reports and analytics
* Seamlessly integrate with tools, such as Katalon Studio, Katalon TestCloud, and Jira.

File: README.md [hunk 6]
```diff
@@ -1,9 +1,42 @@
...
+
+## Community and support
 
 * Read our [documentation](https://docs.katalon.com/katalon-analytics/docs/overview.html).
-* Ask a question on [Forum](https://forum.katalon.com/categories/katalon-analytics).
-* Request a new feature on [GitHub](CONTRIBUTING.md).
+* Learn about test orchestration and DevOps by enrolling in [Katalon Academy](https://academy.katalon.com/#katalon-testops) courses.
+* Ask questions on the [Katalon Community](https://forum.katalon.com/c/katalon-analytics) forum.
+* For community contributions, refer to our [contribution guidelines](CONTRIBUTING.md).
 * Vote for [Popular Feature Requests](https://github.com/katalon-analytics/katalon-analytics/issues?q=is%3Aopen+is%3Aissue+label%3Afeature-request+sort%3Areactions-%2B1-desc).
```
Comment (maile-katalon) [inline README.md:28]: When you are at it, please remove this line as well as we do not follow that process.

File: README.md [hunk 7]
```diff
@@ -1,9 +1,42 @@
...
 * File a bug in [GitHub Issues](https://github.com/katalon-analytics/katalon-analytics/issues).
```
Comment (maile-katalon) [inline README.md:29]: Can we instruct them to post a ticket to the support desk instead of GitHub issues? Single source of tracking and response.

**Comment by maile-katalon** [comment]
Hi @haolekatalon, thank you very much for working on this request. I have added a few comments.
Please let me know if you have any questions.
Thank you.
Mai

File: README.md [hunk 8]
```diff
@@ -25,8 +23,7 @@ For instructions on using Katalon TestOps, visit our documentation website: [Ful
 * Learn about test orchestration and DevOps by enrolling in [Katalon Academy](https://academy.katalon.com/#katalon-testops) courses.
 * Ask questions on the [Katalon Community](https://forum.katalon.com/c/katalon-analytics) forum.
 * For community contributions, refer to our [contribution guidelines](CONTRIBUTING.md).
-* Vote for [Popular Feature Requests](https://github.com/katalon-analytics/katalon-analytics/issues?q=is%3Aopen+is%3Aissue+label%3Afeature-request+sort%3Areactions-%2B1-desc).
-* File a bug in [GitHub Issues](https://github.com/katalon-analytics/katalon-analytics/issues).
+* For bug reports, submit a ticket at [Katalon Support](https://support.katalon.com/hc/en-us).
```
Comment (maile-katalon) [inline README.md:26]: * For bug reports or feature requests, submit a ticket at [Katalon Support](https://support.katalon.com/hc/en-us).

**Pull request merged**